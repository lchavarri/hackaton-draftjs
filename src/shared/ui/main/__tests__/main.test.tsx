import React from 'react';
import { shallow } from 'enzyme';
import Main from '../main';

describe('Main component', () => {
  it('should match component snapshot', () => {
    const wrapper = shallow(<Main />);

    expect(wrapper).toMatchSnapshot();
  });
});
