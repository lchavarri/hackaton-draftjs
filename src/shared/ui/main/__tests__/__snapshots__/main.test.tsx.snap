// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main component should match component snapshot 1`] = `
<ContextProvider
  value={
    Object {
      "appContainerRef": Object {
        "current": null,
      },
      "currentRecord": Object {
        "get": [Function],
        "set": [Function],
      },
      "documentId": "standalone_doc",
      "fullscreen": false,
      "isQuipApp": false,
      "liveAppInstanceId": "standalone",
      "setActionsMenu": [Function],
      "setFullscreen": [Function],
    }
  }
>
  <ActionsMenu
    menu={null}
  >
    <div
      className="app-main main"
    >
      <ApolloApp />
    </div>
  </ActionsMenu>
</ContextProvider>
`;
